### DISCUSSION TITLE
Is there any recommended way to assign different mood pictures to the same character?

### DISCUSSION THREAD

**Discussion by DocMine** [nathanhoad/godot_dialogue_manager#315] [Q&A]
Thank you for the awesome work you have done! 
I need to match different dialogue images for the same character. Is there any good way to mark the image corresponding to this sentence in the dialogue file?
I noticed that there is a variable called "tags" in the "DialogueLine" class. Is it possible to set this variable in the dialog to distinguish the illustrations?
ps: I had imagined writing the speaker as "speaker:Happy" in this form, and then dismantling and parsing the string, but I would like to know if you have a more recommended approach.

**Comment by nathanhoad** [comment]
You could just use something like `[#happy]` and then in your balloon you can see what tags are on a line to swap out the portrait texture based on them.

**Comment by Watercolordevdev** [comment]
> You could just use something like `[#happy]` and then in your balloon you can see what tags are on a line to swap out the portrait texture based on them.

What kind of code checks what tags are on a line? Can you do scripting from inside the dialogue manager to swap things?
